_translator-files/po/de/Running-a-Server.po

```diff
@@ -192,111 +200,115 @@ msgid ""
 "\t<figcaption>How Unregistered Servers work</figcaption>\n"
 "</figure>\n"
 msgstr ""
+"<figure>\n"
+"\t<img src=\"{% include img/en-screenshots/diagram-unreg-server.inc %}\" loading=\"lazy\" alt=\"Diagram of connections between Clients within a Jamulus Unregistered Server\">\n"
+"\t<figcaption>Wie unregistrierte - private Server funktionieren</figcaption>\n"
+"</figure>\n"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:70
 msgid "If you are running an unregistered server behind a home internet connection, you might need to enable [port forwarding](#port-forwarding) as described below."
-msgstr ""
+msgstr "Wenn du einen unregistrierten/privaten Server über eine private Internetverbindung betreibst, musst du möglicherweise die [Portweiterleitung](#port-forwarding) wie unten beschrieben aktivieren."
```

Comment:
#port-forwarding

Reply:
Corrected. Pointing to "Portweiterleitung"